packages/next-slicezone/package-lock.json

```diff
@@ -7061,6 +7061,20 @@
 				"is-typedarray": "^1.0.0"
 			}
 		},
+		"node_modules/typescript": {
```

Comment:
I don't think there should be changes to this file, especially because the package.json hasn't been modified.

But on the other hand, I've seen this change been added in several PRs already so maybe let's pass it once and for all as long as the package isn't published after.

Reply:
good point, I'm not sure why it was added :/

Reply:
removed :)